CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Notable Changes
```

Comment:
Migration notes?

Reply:
Are you suggesting I change this title to Migration Notes? In previous version notes, I found this line under ### Notable Changes so mimicked that. But I can change if you think that's better, I don't have a strong opinion.